pkgs/_macro_tool/README.md

```diff
@@ -1,15 +1,74 @@
-A tool for running macro examples end to end.
+# macro_tool
```

Comment:
nit: in general we don't include these headers, the context should already be obvious